ur_robot_driver/test/trajectory_test.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+import sys
+import time
+import unittest
+
+import rospy
+import actionlib
+from control_msgs.msg import FollowJointTrajectoryAction, FollowJointTrajectoryGoal, FollowJointTrajectoryResult
+from ur_dashboard_msgs.msg import SetModeAction, SetModeGoal, RobotMode
+from std_srvs.srv import Trigger, TriggerRequest
+from trajectory_msgs.msg import JointTrajectoryPoint
+
+PKG = 'ur_rtde_driver'
+NAME = 'trajectory_test'
+
+
+class TrajectoryTest(unittest.TestCase):
+    def __init__(self, *args):
+        super(TrajectoryTest, self).__init__(*args)
+        rospy.init_node('trajectory_testing_client')
+        self.client = actionlib.SimpleActionClient(
+            '/scaled_pos_traj_controller/follow_joint_trajectory', FollowJointTrajectoryAction)
+        timeout = rospy.Duration(30)
+        try:
+            self.client.wait_for_server(timeout)
+        except rospy.exceptions.ROSException as err:
+            self.fail(
+                "Could not reach controller action. Make sure that the driver is actually running."
+                " Msg: {}".format(err))
+
+        self.init_robot()
+
+    def init_robot(self):
+        """Make sure the robot is booted and ready to receive commands"""
+        mode_client = actionlib.SimpleActionClient(
+            '/ur_hardware_interface/set_mode', SetModeAction)
+        timeout = rospy.Duration(30)
+        try:
+            mode_client.wait_for_server(timeout)
+        except rospy.exceptions.ROSException as err:
+            self.fail(
+                "Could not reach set_mode action. Make sure that the driver is actually running."
+                " Msg: {}".format(err))
+        goal = SetModeGoal()
+        goal.target_robot_mode = RobotMode.RUNNING
+        goal.play_program = False # we use headless mode during tests
+
+        mode_client.send_goal(goal)
+        mode_client.wait_for_result()
+
+        self.assertTrue(mode_client.get_result().success)
+
+        send_program_srv = rospy.ServiceProxy("/ur_hardware_interface/resend_robot_program", Trigger)
+        send_program_srv.call()
+        rospy.sleep(5)
+
+    def test_trajectory(self):
+        """Test robot movement"""
+        goal = FollowJointTrajectoryGoal()
+
+        goal.trajectory.joint_names = ["elbow_joint", "shoulder_lift_joint", "shoulder_pan_joint",
+                                       "wrist_1_joint", "wrist_2_joint", "wrist_3_joint"]
+        position_list = [[0.0 for i in range(6)]]
```

Comment:
Is the `in range(6)` construct preferable to:

```suggestion
        position_list = [[0.0]*6]
```
?

Reply:
I personally find it more readable.